packages/grid/src/utils.ts

```diff
@@ -0,0 +1,39 @@
+import { getThemedState, IThemeNamespace } from '@xcritical/theme';
+
+import { gridThemeNamespace, defaultTheme } from './theme';
+import { IGridTheme, IMappedItem } from './interfaces';
+
+
+export const guid = () => {
+  function s4() {
+    return Math.floor((1 + Math.random()) * 0x10000)
+      .toString(16)
+      .substring(1);
+  }
+
+  return `${s4() + s4()}-${s4()}-${s4()}-${s4()}-${s4()}${s4()}${s4()}${s4()
+    + s4()}-${s4()}-${s4()}-${s4()}-${s4()}${s4()}${s4()}`;
+};
+
+export const addOrDeleteItemFromArray = (array: string[], item: string) => {
+  if (array.some((el: string) => el === item)) return array.filter((el: string) => el !== item);
+  return [...array, item];
+};
+
+export function gridTheme<T>(
```

Comment:
Where type 'T' is used in this function? Maybe it should be removed?

Reply:
if delete <T> TS2558: Expected 0 type arguments, but got 1. :(